### PR TITLE
Self Hosting Section

### PR DESCRIPTION
**Pull request by phazonoverload**

### CODE REVIEW

File: content/12.self-hosting/1.overview.md [hunk 1]
```diff
@@ -1,6 +1,46 @@
...
+
+There are many benefits to self-hosting, but there are also some considerations to keep in mind.
+
+### Control
+
+Some users prefer to have full control over their infrastructure and deployment configuration. This means you have the freedom to choose the hardware that best fits your needs. As Directus is provided as a Docker image, you can deploy it on many different platforms.
+
+You can also control all of the required parts for a production system, including the database, caching, and backups. All environment variables are configurable, giving you the ability to customize the behavior of Directus to your needs, even if the desired options are not recommended for security or stability.
+
+### Skills
```
Comment (hola-soy-milk) [inline content/12.self-hosting/1.overview.md:20]: Something about the word "skills" is bothering me. Not that it's condescending but hm. I wonder if we can replace it with something like "maintenance"

File: content/12.self-hosting/2.requirements.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+description: This page outlines the requirements for self-hosting Directus.
+---
+
+# Self-Hosting Requirements
+
+We distribute Directus as a Docker image. To self-host Directus, you will need to be in an environment that supports Docker.
+
+## Variables
+
+Directus uses environment variables to configure many advanced features. In order to run Directus, your host will need to provide the ability to edit the required variables. You can learn more about the available variables in the [configuration section](/configuration/overview).
```
Comment (hola-soy-milk) [inline content/12.self-hosting/2.requirements.md]:
```suggestion
Directus uses environment variables to configure many advanced features. In order to run Directus, your host will need to provide the ability to edit the required variables. You can learn more about the available variables in the [configuration section](/configuration/general).
```

File: content/12.self-hosting/3.deploying.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+description: This section covers the deployment process of Directus, including environment variables, versioning, persistence, and initial admin user setup.
+---
+
+# Deploying Directus
+
+Directus is provided as a Docker image. This means you can deploy it on many different platforms. While each is slightly different, the core concepts are the same.
+
+## Environment Variables
+
+Directus uses environment variables for database and asset storage connection details and some [key project configuration](/configuration/overview). At the very least, you will need to [configure a database](/configuration/database) and set a `SECRET`.
```
Comment (hola-soy-milk) [inline content/12.self-hosting/3.deploying.md]:
```suggestion
Directus uses environment variables for database and asset storage connection details and some [key project configuration](/configuration/general). At the very least, you will need to [configure a database](/configuration/database) and set a `SECRET`.
```

File: content/12.self-hosting/3.deploying.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+Dependent on your hosting provider, you will need to set these variables in a different place. In some, it is a dedicated key/value store provided in a Web UI. In others, it is a file that is loaded when the Docker container starts. As Directus is provided as a Docker image, you shouldn't need to do more than set environment variables to get started.
+
+## Versioning Pinning
```
Comment (hola-soy-milk) [inline content/12.self-hosting/3.deploying.md]: "Versioning Pinning" feels off. Would there be an argument against going with Version Pinning?

**Review by hola-soy-milk — COMMENTED** [review]
Some minor questions, but overall looks solid!

File: content/12.self-hosting/3.deploying.md (hunk 4 above)
Comment (phazonoverload) [inline content/12.self-hosting/3.deploying.md]:
```suggestion
## Version Pinning
```

File: content/12.self-hosting/1.overview.md (hunk 1 above)
Comment (phazonoverload) [reply content/12.self-hosting/1.overview.md:20]: It's a tough one but I think it's the right word. You either have the skills or you don't.

**Pull request merged**